modules/core/test/src/smithy4s/CachedSchemaVisitorSpec.scala

```diff
@@ -25,6 +27,7 @@ class CachedSchemaVisitorSpec() extends FunSuite {
         tag: CollectionTag[C],
         member: Schema[A]
     ): Option[C[A]] = {
+      self(member)
```

Comment:
why not `this(member)`?

Reply:
i just use self instead of this by habit , there is no difference in this case